apps/src/lib/script-editor/LevelTokenDetails.jsx

```diff
@@ -0,0 +1,226 @@
+  },
+
+  render() {
+    return (
+      <div style={styles.levelTokenActive}>
+        <span style={Object.assign({float: 'left'}, styles.levelFieldLabel)}>
+          Level type
+        </span>
+        <VirtualizedSelect
+          value={this.props.level.kind}
+          options={this.levelKindOptions}
+          onChange={this.handleLevelTypeSelected}
+          clearable={false}
+          arrowRenderer={ArrowRenderer}
+          style={styles.levelTypeSelect}
+        />
+        {this.containsLegacyLevel() &&
+          <div>
+            <span
+              style={Object.assign(
+                {display: 'inline-block'},
+                styles.levelFieldLabel
+              )}
+            >
+              Skin
+            </span>
+            <input
+              defaultValue={this.props.level.skin}
+              type="text"
+              style={styles.textInput}
+            />
+            <div style={{float: 'right'}}>
+              <span
+                style={Object.assign(
+                  {display: 'inline-block'},
+                  styles.levelFieldLabel
+                )}
+              >
+                Video key
+              </span>
+              <input
+                defaultValue={this.props.level.videoKey}
+                type="text"
+                style={styles.textInput}
+              />
+            </div>
+            <div style={{clear: 'both'}}></div>
+            <span
+              style={Object.assign(
+                {display: 'inline-block'},
+                styles.levelFieldLabel
+              )}
+            >
+              Difficulty
+            </span>
+            <input
+              defaultValue={this.props.level.conceptDifficulty}
+              type="text"
+              style={styles.textInput}
+            />
+            <div style={{float: 'right'}}>
+              <span
+                style={Object.assign(
+                  {display: 'inline-block'},
+                  styles.levelFieldLabel
+                )}
+              >
+                Concepts
+              </span>
+              <input
+                defaultValue={this.props.level.concepts}
+                type="text"
+                style={Object.assign({}, styles.textInput, {width: 320})}
+              />
+            </div>
+          </div>
+        }
+        {this.props.level.ids.map((id, index) =>
+          <div key={id}>
+            {this.props.level.ids.length > 1 &&
+            <div>
+              <hr style={styles.divider} />
+              <span style={styles.levelFieldLabel}>Active</span>
+              <input
+                type="radio"
+                onChange={this.handleActiveVariantChanged.bind(this, id)}
+                defaultChecked={id === this.props.level.activeId}
+                style={styles.checkbox}
+                name={`radio-${this.props.stagePosition}-${this.props.level.position}`}
+              />
+            </div>
```

Comment:
I think we're a little bit inconsistent about this in our code (and styleguide might even say both are okay), but I think I've come around to agree with brad that it's better in cases like these to indent our JSX when part of a conditional continuation, i.e.
```jsx
{this.props.somethingIsTrue &&
  <MyDiv/>
}
// instead of
{this.props.somethingIsTrue &&
<MyDiv/>
}